README.md

```diff
@@ -1 +1,47 @@
-# to_do_list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To Do List
+
+> This is a to-do list using WebPack.
+
+Additional description about the project and its features.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+##Live Demo
+![live demo link](https://competent-swirles-bcd6c1.netlify.app)
+
+## Getting Started
+
```

Comment:
The project looks awesome! But README should be professional and clear for users. Please add `npm install` and `npm start` commands so a non-technical user can set the project up locally.